src/extension/background-script/router.ts

```diff
@@ -19,6 +19,7 @@ const routes = {
     lnurl: webln.lnurl,
     makeInvoice: webln.makeInvoiceOrPrompt,
     verifyMessage: ln.verifyMessage,
+    getInvoices: ln.invoices,
```

Comment:
this should not be here. getInvoices is currently not a webln call and can not be accessed through webln

Reply:
Removed